### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by lerfast**
Hello fellow code reviewer, we did the following:

 Remove current_user method in ApplicationController, devise will provide us with one.
 [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
 Can register a new user.
 User logs in with a combination of email and password.
 Hashed password should be stored in the database.
 Ask for confirmation of email.
 Can reset password.

### CODE REVIEW

File: app/views/layouts/application.html.erb
Comment (cynthiainga) [inline app/views/layouts/application.html.erb:1]: - [ ] As mentioned in the project requirements the registration and login should be modified to match the wireframes, kindly fix the design 🙃 😁 

Your design

Screenshot 2023-11-08 at 21 48 27

Given wireframe

Screenshot 2023-11-08 at 21 51 05

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @lerfast, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] The `current_user` method in `ApplicationController` has been removed ✔️ 
- [ ] Devise has been installed and setup ✔️ 
- [ ] The application allows for registering a new user ✔️ 
- [ ] A user can log in with a combination of email and password ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/layouts/application.html.erb
Comment (lerfast) [reply app/views/layouts/application.html.erb:1]: I put a lot of effort on this
image

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @lerfast, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Gitflow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**